packages/controllers/profile_edit.coffee

```diff
@@ -0,0 +1,29 @@
+if Meteor.isClient
+  Template.profileEdit.onCreated ->
+    instance = this
+    subscription = instance.subscribe('userProfile')
+
+  Template.profileEdit.helpers
+    userProfile: ->
+      UserProfiles.findOne()
+
+  Template.profileEdit.events
+    'submit form': (event) ->
+      event.preventDefault()
+      form = event.target
+      fields = {
+        fullName: form.fullName?.value
+        jobTitle: form.jobTitle?.value
+        bio: form.bio?.value
+        emailHidden: form.emailHidden?.value
+      }
+      Meteor.call('updateProfile', fields, -> {})
+
+if Meteor.isServer
+  Meteor.methods
+    updateProfile: (fields, callback) ->
```

Comment:
the callback argument to a Meteor method isn't mentioned in the documentation - how does it work?